sql/updates/characters/3.3.5/9999_99_99_99_characters.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `arena_team` ADD `previousOpponents` int(10) unsigned NOT NULL DEFAULT 0;
```

Comment:
Do we really need to store this in the database?
Personally, it seems like an acceptable loss of data in case of a server crash/restart.

Reply:
Well, does it work any better, worse or the same with this PR code added to base TC source?
(presuming that you have tested this with multiple players, of course)

Reply:
@tkrokli: It's certainly useless, I'll eliminate it.